### ISSUE TITLE
After a while gets stuck on simulated follow

### ISSUE THREAD

**Issue by FarkasLabs** [InstaPy/InstaPy#2520]
## Current Behavior
After about 30-40 minutes using the `follow_user_followers` feature the bot simply stops and won't continue. In the first 30-40 minutes everything works fine.

> INFO [2018-07-24 10:40:02] [abc]  User 'xyz' [1/1]
> INFO [2018-07-24 10:40:09] [abc]  Simulated follow : 1

## Expected Behavior
Well, it should keep working just like before...

## InstaPy configuration
in a for loop:
```
session.like_by_tags
session.follow_user_followers
session.unfollow_users
```

**Comment by FarkasLabs** [comment]
It seems to be fixed with this: #2519 
Will monitor it a bit more and close the issue then.

**Comment by FarkasLabs** [comment]
Unfortunately it still happens. Now not everytime but at least once a day it does :(
